packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -217,7 +217,10 @@ export function getSpawnOptions(
   nodeVersion: NodeVersion
 ): SpawnOptions {
   const opts = {
-    env: { ...process.env },
+    env: {
+      PATH: process.env.PATH,
+      ...process.env,
+    },
```

Comment:
**suggestion (non-blocking):** extract to `cloneEnv`

If we extract this into a utility function (like `cloneEnv`), we can add comment on the function explaining why this is necessary.

```
function cloneEnv(currentEnv) {
  return {
    // set uppercase `PATH` for Windows, which would normally happen
    // for us because Node.js proxies the value, but when we enumerate
    // the values below, we lose that proxied value
    PATH: currentEnv.PATH,

    ...currentEnv
  }
}
```

Reply:
I thought of that. Wasn't sure if `build-utils` was the best place or what. I will fix it.

Reply:
Definitely needs a helper function. I think build-utils is the best place for it.